WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -38,14 +38,21 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v18.6-whats-new"
+msgctxt "v18.7-whats-new"
 msgid ""
-"- We've made it easier to see when there are new blocks available for use on your site\n"
-"- You can now highlight text using the <mark> HTML tag, as well as change the text and background color of the Search block button. Shiny!\n"
-"- We've hidden HTML tags in post titles when you're checking out your stats.\n"
-"- If you're part of a comment thread, you can keep track of it through in-app notifications, not just your email inbox.\n"
-"- The Help screen used to break out of its box when rotated from portrait to landscape and back to portrait, but now it's staying put.\n"
-"- We fixed more cases where embed blocks would show "unable to display content", allowing you to see all of your content.\n"
+"Two-step authentication now comes with extra security. You'll need to unlock your device to approve or deny notifications.\n"
+"We expanded the “Link To” setting for the image block to include “Media File” and “Attachment Page” options alongside the preexisting “Custom URL” option.\n"
```

Comment:
For ref: discussed in [WPorg's Slack here](https://wordpress.slack.com/archives/C02RQC4LY/p1637178814019900?thread_ts=1637173139.014900&cid=C02RQC4LY)

Reply:
The link's in the commit message, too 👍